modules/claims_api/app/controllers/concerns/claims_api/v2/json_format_validation.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+module ClaimsApi
+  module V2
+    module JsonFormatValidation
+      extend ActiveSupport::Concern
+
+      included do
+        def validate_json_format
+          # rubocop:disable Style/ClassEqualityComparison
+          # testing string b/c NullIO class doesn't always exist
+          if request.body.class.name == 'Puma::NullIO'
+            render_body_is_not_a_hash_error request.body
+            return
+          end
+          # rubocop:enable Style/ClassEqualityComparison
+
+          @json_body = JSON.parse request.body.read
+          return if @json_body.is_a? Hash
+
+          render_body_is_not_a_hash_error @json_body
+        rescue JSON::ParserError
+          render_body_is_not_a_hash_error request.body.read
+        end
+
+        def render_body_is_not_a_hash_error(body)
+          status = '422'
+          error = {
+            title: 'Inprocessable entity',
+            status:,
+            detail: "The request body isn't a JSON object: #{body.inspect}",
+            source: ''
+          }
+          render status:, json: { errors: [error] }
+        end
+      end
+    end
+  end
+end
```

Comment:
The reason for this change is that the code or status needs to be a string.  In the JSON validator for v1 (previously shared between v1 and v2) the code was being returned as a number.  LH standards directly say it should be a string.  So even though it si a small change I felt that required a v2 version of this file.